src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -69,28 +71,29 @@ def add_exception(self, exception, fault_type, description=None, message=''):
 
     @decorators.suppress_all_exceptions(raise_in_diagnostics=True, fallback_return=None)
     def generate_payload(self):
-        base = self._get_base_properties()
-        cli = self._get_azure_cli_properties()
-
-        user_task = self._get_user_task_properties()
-        user_task.update(base)
-        user_task.update(cli)
-
-        self.events[DEFAULT_INSTRUMENTATION_KEY].append({
-            'name': '{}/command'.format(PRODUCT_NAME),
-            'properties': user_task
-        })
-
-        for name, props in self.exceptions:
-            props.update(base)
-            props.update(cli)
-            props.update({CORRELATION_ID_PROP_NAME: str(uuid.uuid4()),
-                          'Reserved.EventId': str(uuid.uuid4())})
+        if not self.suppress_new_event:
+            base = self._get_base_properties()
+            cli = self._get_azure_cli_properties()
+
+            user_task = self._get_user_task_properties()
+            user_task.update(base)
+            user_task.update(cli)
+
             self.events[DEFAULT_INSTRUMENTATION_KEY].append({
-                'name': name,
-                'properties': props
+                'name': '{}/command'.format(PRODUCT_NAME),
```

Comment:
Do you always want to add `/command` to the end of this?

Reply:
generate_payload() is used to create an event for regular cli commands; which, if I'm not mistaken, should all have the `/command` appended.
I didn't change any of this functionality.